lib/utils/blockchain-txs.ts

```diff
@@ -421,21 +421,24 @@ export const setDomainRecords = async ({
     }
 
     for (let i = 0; i < Object.keys(addresses).length; i++) {
-      const value = Object.values(addresses)[i];
-
+      const [cryptocurrencyName, address] = Object.entries(addresses)[i];
+      const cryptocurrencyToCoinTypeEntry:
+        | [coinName: string, coinType: string]
+        | undefined = Object.entries(cryptocurrencyToCoinType).find(
+        ([key, _]) => key == cryptocurrencyName.toUpperCase()
+      );
+      if (cryptocurrencyToCoinTypeEntry === undefined) {
+        console.error(`coinType code for ${cryptocurrencyName} not found`);
+        continue;
```

Comment:
Is this the correct keyword to be used in here?

Reply:
I think so, the keyword "continue" will keep the "for" loop going and add the other coins to be registered, only the coin that doesn't have coinType code will be discarded, "break" would break out of the entire "for" loop.

Reply:
It makes sense, @lgahdl !